### ISSUE TITLE
Dual Connection docking

### ISSUE THREAD

**Issue by SumGuyTwitches** [SumGuyTwitches/FuelMyShip#7]
Dual Connection Docking 
-----------
what does docking with a dual connection and then trying to refuel with the fake connection do?

**Comment by linuxgurugamer** [comment]
Probalby nothing, only one of the ports is really docked.  You may want to verify that the PAW menu doesn't show up unless both nodes on the docking port are connected

**Comment by SumGuyTwitches** [comment]
When tested what happened was with 2 docking ports connected, nothing was transferred. My theory is simply that it tags everything as both ship and station, and then gets stuck in an infinite loop.
this only happens on the "not actually docked" port, the other port works fine. A simple solution will be to detect the fake ports and remove the button from them. I think the update to not show the button unless docked will fix it.

**Comment by linuxgurugamer** [comment]
Right.  KSP can't have loops in the connections, so even if you see what appears to be two sets of attached docking ports, only one is really docked.  Which means that the other one (the undocked one) should only have a single connection
